sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -325,6 +347,8 @@ class SQLQueryTestSuite extends QueryTest with SharedSparkSession {
         localSparkSession.conf.set(SQLConf.CROSS_JOINS_ENABLED.key, true)
```

Comment:
not related to this PR, but can we remove it here? CROSS JOIN is already enabled by default now.

Reply:
Oh, yes! I'll remove it.